examples/multi-sensor-app/silabs/src/DataModelCallbacks.cpp

```diff
@@ -64,10 +64,9 @@ void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath &
         VerifyOrReturn(attributeId == TemperatureMeasurement::Attributes::MeasuredValue::Id ||
                        attributeId == RelativeHumidityMeasurement::Attributes::MeasuredValue::Id);
 
-        AppEvent event = {
-            .Type    = AppEvent::kEventType_SensorAttributeUpdate,
-            .Handler = AppTask::SensorAttributeUpdateEvent,
-        };
+        AppEvent event = {};
+        event.Type     = AppEvent::kEventType_SensorAttributeUpdate;
+        event.Handler  = AppTask::SensorAttributeUpdateEvent;
```

Comment:
This change isn't necessary.

Reply:
it was needed, that's why the multi-sensor build was failng earlier on the CSA sync branch, when you cherrypicked the changes.

Reply:
Why is it needed? In theory, you should be able to configure the AppEvent directly in the instatiation.